<?xml version="1.0"?>
<launch>

    <node pkg="iris_cobot" type="filter.py" name="filter" output="screen" respawn="true"/>

    <node pkg="iris_cobot" type="ft_theory.py" name="ft_theory" output="screen" respawn="true"/>

    <node pkg="iris_cobot" type="correct.py" name="correct" output="screen" respawn="true"/>

    <node pkg="iris_cobot" type="weight.py" name="weight" output="screen" respawn="true"/>

    <node pkg="iris_cobot" type="time_series_analysis.py" name="time_series_analysis" output="screen" respawn="true"/>

</launch>